pkg/api/defaults-apiserver.go

```diff
@@ -104,6 +104,11 @@ func (cs *ContainerService) setAPIServerConfig() {
 		}
 	}
 
+	if common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.20.0-alpha.1") {
+		defaultAPIServerConfig["--service-account-issuer"] = "kubernetes.default.svc"
```

Comment:
From https://github.com/kubernetes/kubernetes/pull/95896/files#diff-97e779171708fff85d7b1ee3fac01336bf351c39066428e12526d0fd7da5c05cR201, it seems we also need `service-account-key-file`.

Reply:
We actually already pass that config: https://github.com/Azure/aks-engine/blob/master/pkg/api/defaults-apiserver.go#L31